wimm.Secundatives/TimeOfDay.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace wimm.Secundatives
+{
+    /// <summary>
+    /// Represents a time of day.
```

Comment:
Maybe elaborate a bit here. "Represents a time of day between 0000H and 2359H with no attached tz" etc the summary should say enough.

Reply:
bump

Reply:
DoubleBump

Reply:
TrebleBump